src/Helpers/apiCalls.js

```diff
@@ -62,6 +66,10 @@ export function useCreateCollection() {
     });
 }
 
+export function useCreateUser() {
+    return useMutation(createNewUser);
```

Comment:
now that you have a query to GET all users, you will likely want to add an onSuccess for this mutation and invalidate the "allUsers" query to update the data